gpytorch/lazy/diag_lazy_tensor.py

```diff
@@ -8,6 +8,7 @@
 from .non_lazy_tensor import NonLazyTensor
 
 
+# TODO: Subclass DiagLazyTensor from TriangularLazyTensor
```

Comment:
I was going to suggest this :) Any reason it can't be sub-classed now?

Reply:
time and more test failures :) I'll try to get it done though so it's logically more consistent